### DISCUSSION TITLE
Shouldn't schema be a property of the connection configuration object instead of the initOptions object

### DISCUSSION THREAD

**Discussion by mrft** [vitaly-t/pg-promise#823] [Ideas]
I guess this is a question for @vitaly-t

Why is the 'schema' a setting of the initOptions object and not the connection configuration object?

I mean: with a single instance of the library, I could connect to multiple databases, and I think it only makes sense to specificy which default schema to use at the connection(pool) level, and not at the library level.

What are your thoughts on this?

**Comment by vitaly-t** [comment]
You can create a separate initialization object for each separate database connection, and set `schema` accordingly. Or, alternatively, you can use `dc` to set the right schema on from the same initialization object. It will work either way.

**Reply by mrft** [reply]
I am aware of that, but I posted this in "ideas" because I feel that the initialization object is logically not the right place to put it.

**Reply by vitaly-t** [reply]
The library was written awhile ago, and it has a huge user base. To be changing things like that, I simply do not have the time anymore.
